src/utils/is-url.js

```diff
@@ -1,5 +1,5 @@
 const isURL = require('is-url');
 
 module.exports = function (url) {
-  return isURL(url) || /^#/.test(url);
+  return isURL(url) || /^#/.test(url) || /^[a-z]*\:/i.test(url);
```

Comment:
Could we hoist these regexes into constants and add descriptions?

Reply:
+1

Reply:
@thejameskyle Done